Assets/GothicVR/Scripts/Globals/Hero.cs

```diff
@@ -0,0 +1,31 @@
+using System.Linq;
+using GVR.Caches;
+using GVR.Properties;
+using UnityEngine.SceneManagement;
+using ZenKit.Daedalus;
+
+namespace GVR.Globals
+{
+    public static class Hero
+    {
+        public static void LoadHeroVM()
```

Comment:
Hero has his own VM? Nice. ;-)
```suggestion
        public static void LoadHero()
```